### ISSUE TITLE
JSX and TSX

### ISSUE THREAD

**Issue by pauleveritt** [11ty/eleventy-plugin-template-languages#12]
Hey Zach, looking for any contributors? I wouldn't mind pitching in to add JSX and TSX and languages, as well as general maintenance. I could also help organize a group that pitches in.

I think JSX and TSX should be separate as TSX adds some TypeScript pushups that will frustrate people.

**Comment by zachleat** [comment]
Yeah, absolutely! Be awesome to simplify the docs on these pages (or at least provide simplified instructions): https://www.11ty.dev/docs/languages/jsx/ https://www.11ty.dev/docs/languages/typescript/

**Comment by pauleveritt** [comment]
@zachleat Here's a [PR in a fork](https://github.com/pauleveritt/eleventy-plugin-template-languages/pull/1) with tests and writeup.

I propose I toot about it, you quote-toot it it, and see if we can get some non-Zach eyeballs on it.

Once settled, merged, and released, I can do a PR for docs changes.

**Comment by uncenter** [comment]
Why not PR it to this repository?

**Comment by pauleveritt** [comment]
@uncenter I did, [yesterday](https://github.com/11ty/eleventy-plugin-template-languages/pulls).

**Comment by uncenter** [comment]
Did you mean to open two duplicate PRs?
![CleanShot 2024-07-31 at 17 18 25@2x](https://github.com/user-attachments/assets/0bdd7516-1080-4c22-b412-5fbf412c5870)

**Comment by pauleveritt** [comment]
I did, I wanted to treat JSX as different from TSX. The latter has a bit more ceremony (and Zach is doing TS stuff ATM.)

If you'd prefer, I can combine them.
